.circleci/config.yml

```diff
@@ -632,12 +633,32 @@ commands:
             - "3b:cd:47:bf:57:9c:e5:36:b0:4d:5f:12:5e:d3:b3:3e"
 
   install-mbx-ci:
+    parameters:
+      platform:
+        type: enum
+        enum:
+          - darwin
+          - linux
+        default: darwin
+      arch:
+        type: enum
+        enum:
+          - amd64
+          - arm64
+        default: amd64
+      version:
+        type: string
+        default: latest
+      install_path:
+        type: string
+        default: /usr/local/bin
     steps:
       - run:
           name: Install latest mbx-ci
           command: |
-            curl -Ls https://mapbox-release-engineering.s3.amazonaws.com/mbx-ci/latest/mbx-ci-darwin-amd64 > mbx-ci && chmod 755 ./mbx-ci
-            ./mbx-ci aws setup
+            curl -Ls https://mapbox-release-engineering.s3.amazonaws.com/mbx-ci/<<parameters.version>>/mbx-ci-<<parameters.platform>>-<<parameters.arch>> > <<parameters.install_path>>/mbx-ci
+            chmod 755 <<parameters.install_path>>/mbx-ci
```

Comment:
Not convinced that the added complexity is worth it if we always use the defaults at the moment. Do you foresee a future in which we'll use this feature?

Reply:
It's a template for future reuse orbs and so on. Let's keep it. Defaults are fine for Maps-iOS usage now.